src/java.desktop/share/classes/javax/swing/plaf/synth/SynthGraphicsUtils.java

```diff
@@ -665,9 +670,17 @@ static void paintText(Graphics g, SynthMenuItemLayoutHelper lh,
                 g.setColor(lh.getStyle().getColor(
                         lh.getContext(), ColorType.TEXT_FOREGROUND));
                 g.setFont(lh.getStyle().getFont(lh.getContext()));
+
+                int mnemIndex = lh.getMenuItem().getDisplayedMnemonicIndex();
+                // Check to see if the Mnemonic should be rendered in GTK.
+                if (UIManager.getBoolean("RootPane.altPress")
+                    && SynthLookAndFeel.isMnemonicHidden() == true) {
```

Comment:
```suggestion
                if (UIManager.getBoolean("RootPane.altPress")
                    && SynthLookAndFeel.isMnemonicHidden()) {
```

Functionally the same.

Reply:
Updated.